doc/source/index.rst

```diff
@@ -32,21 +32,21 @@ it is now a collection of many Python packages for using Ansys products through
 
       Pythonic interface to PyAnsys Math libraries
 
-    .. grid-item-card:: PyDPF-Core
+    .. grid-item-card:: PyDPF-Post
       :img-top: _static/thumbnails/pydpf-core.png
-      :link: https://dpf.docs.pyansys.com
+      :link: https://post.docs.pyansys.com/
       :text-align: center
       :class-title: pyansys-card-title
 
-      Pythonic interface to DPF (Data Processing Framework) for building more advanced and customized workflows
+      Pythonic interface to DPF (Data Processing Framework) for quick post-processing of simulation data
 
-    .. grid-item-card:: PyDPF-Post
+    .. grid-item-card:: PyDPF-Core
       :img-top: _static/thumbnails/pydpf-core.png
-      :link: https://post.docs.pyansys.com/
+      :link: https://dpf.docs.pyansys.com
       :text-align: center
       :class-title: pyansys-card-title
 
-      Pythonic interface to DPF's postprocessing toolbox for manipulating and transforming simulation data
+      Pythonic interface to DPF (Data Processing Framework) for building more advanced and customized workflows
```

Comment:
Can we switch the order? Just to keep them in alphabetical order =)

Reply:
The thing is that it looks "odd" to find these two packages disordered whi8le the rest of them are. I'd suggest that promotion of one over the other should be done inside each of the libraries/docs rather than here. All entries appear as cards on a matrix view, so it's not like they are listed out in order. They would appear side by side.

Reply:
Thanks @PProfizi, really appreciated!!